trackthenews/core.py

```diff
@@ -427,9 +427,6 @@ def main():
                 deduped.append(article)
 
         for counter, article in enumerate(deduped, 1):
-            # aquire exclusive access here since changes are commited later int he loop
-            conn.execute('BEGIN EXCLUSIVE')
```

Comment:
Was this removed on purpose? Did it cause the issue?

Reply:
It was on purpose, and I believe it caused the issue in 3.6 in particular (or, at least, removing this one but not the one on L367 seemed to fix it). I am not totally clear on the sqlite isolation levels but I suspect the issue might have stemmed from a [change in the behavior in 3.6](https://docs.python.org/3.6/library/sqlite3.html#sqlite3-controlling-transactions) where there is no longer an implicit "open" statement?